### ISSUE TITLE
Temp. admin nickname color under the !players list on IRC

### ISSUE THREAD

**Issue by BeanerLVP** [LVPlayground/playground#533]
With the new Nuwani bot, temporary admins appear to have the same red color as other crew members under the !players list. I suggest we either pick another color for them, or simply go back to using level-based colors (black for regulars, blue for VIPs).

If point 6 is implemented from this ticket: https://github.com/LVPlayground/playground/issues/532, we don't need them to have the crew color under !players anyway. This makes it easier for players to distinguish between temporary and permanent crew, and approach them accordingly depending on the nature of assistance they require.

It's obviously a minor/low-priority feature, but I thought I'd bring it up since even I was confused earlier and wondered if we promoted someone today.

**Comment by Specifer** [comment]
@BeanerLVP , I think showing temp-admins in-game as red by !players would probably be a good feature since you don't have to call other admins from IRC as a temp is there . And !admins command works for only admins , so this paves a way for players to know who is temp right at that time .

**Comment by RussellLVP** [comment]
I have mixed feelings about this. If we trust someone to have temporary administrator rights without anyone being directly available to help them out, surely the current reflection in `!players` is, in fact, accurate?

**Comment by BeanerLVP** [comment]
> 
> 
> I have mixed feelings about this. If we trust someone to have temporary administrator rights without anyone being directly available to help them out, surely the current reflection in `!players` is, in fact, accurate?

It is accurate to some extent, but temporary administrators are limited to having session-based **in-game** rights only. This can mislead players on IRC if there isn't any distinction.

Some players tend to ignore the little details. Let's say there's an old player returning to LVP who lost their forum password: they might not know if the player in-game is a crew member or a temp. admin, and might proceed with sharing certain details about their bans; discussions with other crew members; or try  to appeal, etc — information that temp. admins don't have access to.

> Is this person a crew member who can help me with account-related settings? 

> Can I share my IP or serial with this person and ask for an unban via IRC?

> He appeared to be an admin yesterday! Did he get demoted?

Similar questions might arise and lead to further confusion. I imagine it'd also be inconvenient for players to be asked to find permanent crew members each time they have questions like these.

The short version is: 
(1) I feel it's best we display the player's regular level (registered / VIP).
(2) If !ingamecrew gets implemented (issue #532), an additional notice under !players may simplify things and make them look accurate:
```
<Gunther>!players
<Nuwani> Online players (12): (R2D)Franco, (R2D)Mosto, BlooF, Eagle_Force_One, George, Outkast, RTFM, VeinsR, [BA]TheMightyQ, [L4sH0sH]LikeG6, [LikeG6]L4sh_, leaty.tHO
<Nuwani> Use !ingamecrew to view which crew members & temporary administrators are available.
```

**Comment by BeanerLVP** [comment]
As discussed in #598, let's close this.
